test/cli/test/bun-test.test.ts

```diff
@@ -6,6 +6,53 @@ import { describe, test, expect } from "bun:test";
 import { bunExe, bunEnv } from "harness";
 
 describe("bun test", () => {
+  test("bun test include", () => {
+    const cwd = createTest([
+      {
+        filename: "hello.lol.js",
+        contents: /* typescript */ `
+        import { test, expect } from "bun:test"
+        test("test #1", () => {
+          expect(true).toBe(true);
+        });
+        `,
+      },
+      {
+        filename: "nice.lol.ts",
+        contents: /* typescript */ `
+        import { test, expect } from "bun:test"
+        test("test #2", () => {
+          expect(true).toBe(true);
+        });
+        `,
+      },
+      {
+        filename: "index.js",
+        contents: /* typescript */ `
+        import { test, expect } from "bun:test"
+        test("test #3", () => {
+          expect(true).toBe(true);
+        });
+        `,
+      },
+      {
+        filename: "index.lol.ts",
+        contents: /* typescript */ `
+        import { test, expect } from "bun:test"
+        test("test #4", () => {
+          expect(true).toBe(true);
+        });
+        `,
+      },
+    ]);
+    const stderr = runTest({
+      args: ["--include", "**/*.lol.{ts,js}"],
```

Comment:
This test should include files which are in `node_modules` and also in folders. It should also include files which are symlinks to other tests, just to see what happens